misc/test_site/mac_addrs/9a02571e8f01/module_config.json

```diff
@@ -24,13 +24,81 @@
   "servers": {
     "tcp": {
       "ports": {
+        "20": {
```

Comment:
this port list is specific to this device -- if it's for testing purposes then it should be a much shorter list, if it's meant to be a template or default list, then it should be somewhere else... ?

Reply:
Yes, it's for testing purposes, but I decided to include more ports to show that these ports are tested and then not found.
Do you think I should change it then to a list of ports that includes only a passing case and a failing case?

Reply:
Things here should be the minimal subset to 1) be an example of what needs to be done, 2) have all the necessary cases for testing.  Anything else (exhaustive list) should be housed elsewhere (e.g. the qualification/ section) since it's application specific at that point.

Reply:
Things here should be the minimal subset to 1) be an example of what needs to be done, 2) have all the necessary cases for testing.  Anything else (exhaustive list) should be housed elsewhere (e.g. the qualification/ section) since it's application specific at that point.

Reply:
Things here should be the minimal subset to 1) be an example of what needs to be done, 2) have all the necessary cases for testing.  Anything else (exhaustive list) should be housed elsewhere (e.g. the qualification/ section) since it's application specific at that point.

Reply:
OK, I'll reduce the list of ports adhering to your guidance.
The qualification folder already includes examples with a more detailed list of ports.